scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -96,7 +96,8 @@
 	subject	Other	D12	13											
 	publicationIDType	ark		0											
 	publicationIDType	arXiv		1											
-	publicationIDType	bibcode		2											
+	publicationIDType	bibcode		2								
+	publicationIDType	cstr		17											
```

Comment:
The numbers here are the display order so they should be unique. If the intent is to have cstr display 4th in the list, it should be 4 and all the rest should be incremented by 1.

Reply:
Adjusted sequence number